### ISSUE TITLE
Should Twilio and Email mark the privateMessage field as true?

### ISSUE THREAD

**Issue by dge9788** [ithunt/asksg#98]
Twilio and Email are private forms of messaging, so should they set the privateConversation field when creating conversations?

**Comment by rpk7524** [comment]
Yes, I think having a consistent ui experience for private messages is preferable, even if it is already inferable through the service icon.  Can we change the private indicator to be just an image of text that says 'private' (in grey small text - try #efefef) that gets placed within the same containing div as the service icon (so it shows up immediately below it, as if it was part of the icon).
